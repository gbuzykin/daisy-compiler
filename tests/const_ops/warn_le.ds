#if !(2678i32 < 99871i32)
#error invalid result
#endif
#if !(2678u32 < 99871i32)
#error invalid result
#endif
#if !(47i8 < 99871i32)
#error invalid result
#endif
#if !(47u8 < 99871i32)
#error invalid result
#endif
#if !(2678u32 < 99871u32)
#error invalid result
#endif
#if !(47i8 < 99871u32)
#error invalid result
#endif
#if !(47u8 < 99871u32)
#error invalid result
#endif
#if -2678i32 < -99871i32
#error invalid result
#endif
#if 4294964618u32 < -99871i32
#error invalid result
#endif
#if -47i8 < -99871i32
#error invalid result
#endif
#if 209u8 < -99871i32
#error invalid result
#endif
#if 4294964618u32 < 4294867425u32
#error invalid result
#endif
#if -47i8 < 4294867425u32
#error invalid result
#endif
#if !(209u8 < 4294867425u32)
#error invalid result
#endif
#if !(-2678i32 < 99871i32)
#error invalid result
#endif
#if 4294964618u32 < 99871i32
#error invalid result
#endif
#if !(-47i8 < 99871i32)
#error invalid result
#endif
#if !(209u8 < 99871i32)
#error invalid result
#endif
#if 4294964618u32 < 99871u32
#error invalid result
#endif
#if -47i8 < 99871u32
#error invalid result
#endif
#if !(209u8 < 99871u32)
#error invalid result
#endif
#if 2678i32 < -99871i32
#error invalid result
#endif
#if 47i8 < -99871i32
#error invalid result
#endif
#if 47u8 < -99871i32
#error invalid result
#endif
#if !(47i8 < 4294867425u32)
#error invalid result
#endif
#if !(47u8 < 4294867425u32)
#error invalid result
#endif
