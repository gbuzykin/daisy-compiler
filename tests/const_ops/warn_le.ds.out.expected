./const_ops/warn_le.ds:4:15: warning: signed/unsigned mismatch
 4 | #if !(2678u32 < 99871i32)
   |               ^
./const_ops/warn_le.ds:10:12: warning: signed/unsigned mismatch
 10 | #if !(47u8 < 99871i32)
    |            ^
./const_ops/warn_le.ds:16:12: warning: signed/unsigned mismatch
 16 | #if !(47i8 < 99871u32)
    |            ^
./const_ops/warn_le.ds:25:19: warning: signed/unsigned mismatch
 25 | #if 4294964618u32 < -99871i32
    |                   ^
./const_ops/warn_le.ds:31:11: warning: signed/unsigned mismatch
 31 | #if 209u8 < -99871i32
    |           ^
./const_ops/warn_le.ds:37:11: warning: signed/unsigned mismatch
 37 | #if -47i8 < 4294867425u32
    |           ^
./const_ops/warn_le.ds:46:19: warning: signed/unsigned mismatch
 46 | #if 4294964618u32 < 99871i32
    |                   ^
./const_ops/warn_le.ds:52:13: warning: signed/unsigned mismatch
 52 | #if !(209u8 < 99871i32)
    |             ^
./const_ops/warn_le.ds:58:11: warning: signed/unsigned mismatch
 58 | #if -47i8 < 99871u32
    |           ^
./const_ops/warn_le.ds:70:10: warning: signed/unsigned mismatch
 70 | #if 47u8 < -99871i32
    |          ^
./const_ops/warn_le.ds:73:12: warning: signed/unsigned mismatch
 73 | #if !(47i8 < 4294867425u32)
    |            ^
./const_ops/warn_le.ds: info: warnings 11, errors 0
